commands/publish/README.md

```diff
@@ -140,6 +141,16 @@ If you are using a third-party registry that does not support `npm access ls-pac
 
 > Please use with caution
 
+### `--otp`
+
+When publishing to a registry that requires two-factor authentication, you can specify a "one-time password" using `--otp`:
+
+```sh
+lerna publish --otp 123456
+```
+
+> Please keep in mind that one-time passwords often expire within a few minutes of their generation. Please ensure adequate time for the publish process to complete.
```

Comment:
They actually expire within 30 seconds, so it's not clear that this solution would be guaranteed to work in all cases (imagine per-package builds that take 2 seconds each in a repo with 30 packages to publish...).

Reply:
While this is true, until lerna has true OTP support, the only alternative is to set the NPM_CONFIG_OTP environment variable (which differs per system) and has the same downsides.